### PR TITLE
Add helpers to all selectors

### PR DESCRIPTION
**Pull request by balloob**
## Proposed change
Add helper support to all selectors.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-input-helper-text.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@customElement("ha-input-helper-text")
+class InputHelperText extends LitElement {
+  protected render(): TemplateResult {
+    return html`<slot></slot>`;
+  }
+
+  static styles = css`
+    :host {
+      display: block;
+      color: var(--mdc-text-field-label-ink-color, rgba(0, 0, 0, 0.6));
```
Comment (zsarnett) [inline src/components/ha-input-helper-text.ts:13]:
```suggestion
      color: var(--secondary-text-color, rgba(0, 0, 0, 0.6));
```

Should we use this?

**Review by zsarnett — APPROVED** [review]

**Comment by spacegaier** [comment]
Should this really still go into the beta/release? Doesn't really look like a bug fix 😉 and is a rather big change.

File: src/components/ha-input-helper-text.ts (hunk 1 above)
Comment (balloob) [reply src/components/ha-input-helper-text.ts:13]: Right now I have taken the exact variables as used by the `ha-textfield` component.

**Comment by zsarnett** [comment]
Yes. It's needed for all of the config flow changes. Fixes a bug that some of them had it and some didn't 😉

**Pull request merged**